packages/neon/neon_notifications/lib/pages/main.dart

```diff
@@ -28,12 +28,17 @@ class _NotificationsMainPageState extends State<NotificationsMainPage> {
         stream: bloc.notifications,
         builder: (final context, final notifications) => Scaffold(
           resizeToAvoidBottomInset: false,
-          floatingActionButton: FloatingActionButton(
-            onPressed: () async {
-              bloc.deleteAllNotifications();
+          floatingActionButton: StreamBuilder<int>(
+            stream: bloc.unreadCounter,
+            initialData: bloc.unreadCounter.value,
```

Comment:
Please remove this line.

    1. The StreamBuilder will use the current value automatically. No need to provide it manually
    2. The subject in question is not seeded thus it is not guaranteed that it even has a value